README.md

```diff
@@ -25,7 +25,7 @@ compatibility list:
 
 | wasmd | cosmwasm-vm | cosmwasm-std |
 | ----- | ----------- | ------------ |
-| 0.17  | 0.15        | 0.15         |
+| 0.18  | 0.15-0.16   | 0.15-0.16    |
```

Comment:
It is not both. It will be one or the other.
The older versions actually supported many versions of cosmwasm-std simultaneously (as we kept the wasm-rust interface constant)

Reply:
I had added the updated to v0.15 already https://github.com/CosmWasm/wasmd/pull/548/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R28

Reply:
Closing the PR!